### PR TITLE
Man 1172 styling change for custom pages and throughout the site

### PR DESCRIPTION
**Pull request by cdoyle-temple**
Adds the alerts

### CODE REVIEW

File: app/assets/javascripts/listeners.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ $(document).on('turbolinks:load', function() {
 });
 
 $(document).ready(function(){
-
+  // wpvi
```
Comment (sensei100) [inline app/assets/javascripts/listeners.js:36]: @cdoyle-temple I'm not familiar with wpvi.  What does this mean?

**Review by sensei100 — COMMENTED** [review]
@cdoyle-temple Will any of the header-alert updates affect Library Search?

File: app/assets/javascripts/listeners.js (hunk 1 above)
Comment (cdoyle-temple) [reply app/assets/javascripts/listeners.js:36]: This section is for the WPVI videos that we host in the site.

**Comment by cdoyle-temple** [comment]
None of the styling I've done here will affect the header alerts display or functionality in library search.

**Comment by sensei100** [comment]
@cdoyle-temple The alert changes that I was referring to don't seem to be styling changes.  They are updates in the controllers.

File: app/assets/javascripts/listeners.js (hunk 1 above)
Comment (cdoyle-temple) [reply app/assets/javascripts/listeners.js:36]: I believe the alerts you look for in the JSON are Alert.where(published: true).where(for_header: false)

Nothing about that has changed.

**Comment by cdoyle-temple** [comment]
@sensei100 I believe the alerts you look for in the JSON are Alert.where(published: true).where(for_header: false)

Nothing about that has changed.

**Review by sensei100 — APPROVED** [review]

**Pull request merged**